candi/openapi/doc-ru-node_group.yaml

```
@@ -238,18 +238,18 @@ spec:
                         Режим выдачи разрешения на disruptive-обновление:
                         - `Manual` — отключить автоматическую выдачу разрешений на disruption. Когда disruption потребуется — загорится специальный алерт;
                         - `Automatic` —  автоматически выдавать разрешения на disruption при обновлении (значение по умолчанию).
-                    automatic:
+                    automatic: &automatic
                       description: |
                         Дополнительные параметры для режима `Automatic`.
                       properties:
                         drainBeforeApproval:
                           description: |
                             Выгон (draining) Pod'ов с узла перед выдачей разрешения на disruption.
-                            
+
                             **Внимание!** Данная настройка игнорируется (узлам будет выдано разрешение без предварительного выгона Pod'ов с узлов):
                             - для nodeGroup `master` с единственным узлом;
                             - для [выделенного под запуск Deckhouse](https://deckhouse.io/ru/documentation/v1/deckhouse-faq.html#%D0%BA%D0%B0%D0%BA-%D0%B7%D0%B0%D0%BF%D1%83%D1%81%D0%BA%D0%B0%D1%82%D1%8C-deckhouse-%D0%BD%D0%B0-%D0%BF%D1%80%D0%BE%D0%B8%D0%B7%D0%B2%D0%BE%D0%BB%D1%8C%D0%BD%D0%BE%D0%BC-%D1%83%D0%B7%D0%BB%D0%B5) узла, если этот узел в группе узлов единственно рабочий (Ready).
```

```suggestion
                            - для [выделенного под запуск Deckhouse](https://deckhouse.io/ru/documentation/v1/deckhouse-faq.html#как-запускать-deckhouse-на-произвольном-узле) узла, если этот узел в группе узлов единственно рабочий (Ready).
```
However, what will be the link for the cluster doc installation? It seems we need to ask @z9r5 for help there.